src/main/webapp/Portal.html

```diff
@@ -145,6 +145,7 @@
 			head.load("https://cdn.plot.ly/plotly-1.54.5.min.js");
 			head.load("https://unpkg.com/react-plotly.js@2.2.0/dist/create-plotly-component.min.js");
 			head.load(cdnEndpoint + "generated/react-jsonschema-form.js");
+			head.load(cdnEndpoint + "generated/react-router-dom.min.js");
```

Comment:
SRC load failed before this change (ReviewerDashboard requires react-router-dom).